stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -1247,85 +1090,6 @@ function naivesub!(A::UnitLowerTriangular, b::AbstractVector, x::AbstractVector
     end
     x
 end
-# in the following transpose and conjugate transpose naive substitution variants,
-# accumulating in z rather than b[j] significantly improves performance as of Dec 2015
```

Comment:
This is my only concern. But I don't understand which interface the methods below follow to begin with. Our generic 3-arg `ldiv!` has
```julia
ldiv!(Y::AbstractArray, A::AbstractMatrix, B::AbstractArray) = ldiv!(A, copyto!(Y, B))
```
the output first, not last. Without having the `Triag.data` transposed, the same `ldiv!` doesn't even work.

Reply:
Ok, now I'm sure that the signature goes against the `ldiv!` convention. I'm trying to understand know if having these methods brings a performance benefit. If it does, I wonder why we don't have corresponding `rdiv!` analogues.

Reply:
I agree that the argument order here isn't right. Actually, I believe these should just be two argument methods.

Reply:
Regarding `rdiv!` then I think it's just that `ldiv!` is more common than `rdiv!`. It would good to have `rdiv!` versions as well.